docs/modules/ROOT/pages/subscriptions/filtering.adoc

```diff
@@ -177,4 +187,525 @@ movieUpdate(where: {
+
+Considering the following type definitions:
+[source, graphql, indent=0]
+----
+type Movie {
+    title: String
+    genre: String
+    actors: [Actor!]! @relationship(type: "ACTED_IN", properties: "ActedIn", direction: IN)
+}
+interface ActedIn @relationshipProperties {
+    screenTime: Int!
+}
+type Actor {
+    name: String
+}
+----
+
+The format of the `where` argument is:
+[source, graphql, indent=0]
+----
+{
+    movie: {
+        # top-level properties of the node targeted for the subscription operation, supports operators
+        title_IN: []
+    },
+    createdRelationship: {
+        actors: { # field name corresponding to a relationship in the type definition of the node targeted for the subscription operation
+            edge: {
+                 # properties of the relationship, supports operators
+                screenTime_GT: 10,
+            },
+            node: {
+                # top-level properties of the node on the other end of the relationship, supports operators
+                name_STARTS_WITH: ""
+            }
+        }
+    }
+}
+----
+
+Below are some example of how filtering can be applied when creating a subscription to relationship events.
+
+=== Create Relationship
+The following example filters the subscriptions to newly created relationships that are connecting a `Movie` from genres other than "Drama", to an `Actor` with a screen time bigger than 10:
+
+[source, graphql, indent=0]
+----
+movieRelationshipCreated(where: { movie: { genre_NOT: "Drama" }, createdRelationship: { actors: { edge: { screenTime_GT: 10 } } } }) {
+    movie {
+        title
+    }
+    createdRelationship {
+        actors { 
+            screenTime
+            node { 
+                name
+            }
+        }
+    }
+}
+----
+
+NOTE: `Where` will only filter by properties set at the moment of creation.
+
+=== Delete Relationship
+The following example filters the subscriptions to deleted relationships that were connecting a `Movie` of genre Comedy or Adventure to an `Actor` named "Jim Carrey":
+
+[source, graphql, indent=0]
+----
+movieRelationshipDeleted(where: { movie: { genre_IN: ["Comedy", "Adventure"] }, createdRelationship: { actors: { node: { name: "Jim Carrey" } } } }) {
+    movie {
+        title
+    }
+    deletedRelationship {
+        actors { 
+            screenTime
+            node { 
+                name
+            }
+        }
+    }
+}
+----
+
+NOTE: `Where` will only filter by existing properties right before deletion.
+
+
+=== Relationship-related filters
+So far we've seen how to filter the subscription events by using operations on nodes or relationship properties. The relationship-related filtering logic is even more powerful, as filters can also express the expected relationship field, or the expected concrete type at the other end of the relationship when having relationships to Abastract Types.
```

Comment:
I believe it is part of Neo4j's documentation style guide not to use "we" or "us", generally preferring the third person.

```suggestion
In addition to filtering on node or relationship properties, the relationship-related filtering logic is even more powerful, as filters can also express the expected relationship field, or the expected concrete type at the other end of the relationship when having relationships to Abstract Types.
```